tarok/test/cards_tests.cpp

```diff
@@ -53,9 +54,31 @@ TEST_F(CardsTests, TestCountCards) {
   std::iota(all_card_actions.begin(), all_card_actions.end(), 0);
   EXPECT_EQ(tarok::CardPoints(all_card_actions, deck), 70);
   EXPECT_EQ(tarok::CardPoints({}, deck), 0);
-  EXPECT_EQ(tarok::CardPoints({13, 20, 34}, deck), 6);
-  EXPECT_EQ(tarok::CardPoints({13, 20, 34, 36}, deck), 9);
-  EXPECT_EQ(tarok::CardPoints({13, 20, 34, 36, 53}, deck), 14);
+
+  auto cards = {ActionFromCardName("XIV", deck)};
+  EXPECT_EQ(tarok::CardPoints(cards, deck), 0);
+
+  cards = {ActionFromCardName("Mond", deck)};
+  EXPECT_EQ(tarok::CardPoints(cards, deck), 4);
+
+  cards = {ActionFromCardName("Mond", deck),
+           ActionFromCardName("Jack of Diamonds", deck)};
+  EXPECT_EQ(tarok::CardPoints(cards, deck), 6);
+
+  cards = {ActionFromCardName("XIV", deck), ActionFromCardName("Mond", deck),
+           ActionFromCardName("Jack of Diamonds", deck)};
+  EXPECT_EQ(tarok::CardPoints(cards, deck), 6);
+
+  cards = {ActionFromCardName("XIV", deck), ActionFromCardName("Mond", deck),
+           ActionFromCardName("Jack of Diamonds", deck),
+           ActionFromCardName("Queen of Diamonds", deck)};
+  EXPECT_EQ(tarok::CardPoints(cards, deck), 9);
+
+  cards = {ActionFromCardName("XIV", deck), ActionFromCardName("Mond", deck),
```

Comment:
Maybe if `ActionFromCardName` would take a vector of strings (of card names) and return a vector of actions this would make the tests even more readable. This is just a suggestion so feel free to ignore :).

Reply:
Added CardLongNamesToActions that takes vector of strings and returns a vector of actions. Implementation is not really optimised, since it uses CardLongNameToAction which loops over the deck for each card to find its index. Nevertheless this is only used for testing so I believe this shouldn't be a problem.